templates/shadow.hbs

```diff
@@ -1,2 +1 @@
-{{! drawerView.js will appends this on document's body}}
-<div id="shadow" class="shadow js-shadow u-display-none"></div>
+
```

Comment:
@oliverfoster Can this template be removed completely? Is this for backwards compatibility?

Reply:
I can't think of a use case, and we can always use pseudo selectors to add design elements via CSS. Definitely not something we have customized in the past. Not sure what anyone else thinks?

Reply:
I've not amended the shadow template previously but I wonder if it would be useful to keep. With the dialog animation being updated to use CSS animate this gives us more flexibility to be creative whereas we've typically not touched this. Also, the notifyPopup template does include the [shadow](https://github.com/adaptlearning/adapt-contrib-core/blob/a1b6c57f8608d0a38ef6918ce3bc9bad6961f580/templates/notifyPopup.hbs#L106) so I think from a consistency prospective we should maintain both or remove both.

Reply:
Thanks, @kirsty-hames . Let's leave it then.

Reply:
Now the pressure is on to do something funky with it 💃